frontend/src/metabase/dashboard/actions/data-fetching.js

```diff
@@ -238,14 +238,9 @@ export const fetchDashboard = createThunkAction(
 
       const parameterValuesById = preserveParameters
         ? getParameterValues(getState())
-        : getParameterValuesByIdFromQueryParams(
-            parameters,
-            queryParams,
-            metadata,
-            {
-              forcefullyUnsetDefaultedParametersWithEmptyStringValue: true,
-            },
-          );
+        : getParameterValuesByIdFromQueryParams(parameters, queryParams, {
```

Comment:
`getParameterValuesByIdFromQueryParams` didn't use its `metadata` arg, so I removed it.